ts/store/actions/backendInfo.ts

```diff
@@ -2,16 +2,15 @@
  * Action types and action creator related to BackendInfo.
  */
 import { ActionType, createStandardAction } from "typesafe-actions";
-
-import { ServerInfo } from "../../../definitions/backend/ServerInfo";
+import { IOVersionInfo } from "../reducers/backendInfo";
 
 export const backendInfoLoadFailure = createStandardAction(
   "BACKEND_INFO_LOAD_FAILURE"
 )<Error>();
 
 export const backendInfoLoadSuccess = createStandardAction(
```

Comment:
```suggestion
export const versionInfoLoadSuccess = createStandardAction(
```
maybe it describes better the new data

Reply:
I'm working on renaming and refactoring in https://pagopa.atlassian.net/browse/IAI-122, if isn't a problem I would concentrate all the renaming part there